modules/database/src/main/java/com/intuit/wasabi/database/impl/DBITransaction.java

```diff
@@ -41,9 +42,8 @@
 public class DBITransaction implements Transaction {
 
     private static final Logger LOGGER = getLogger(DBITransaction.class);
-    private Handle handle;
     private DBI dbi;
-    private boolean inTransaction = false;
+    private AtomicBoolean inTransaction = new AtomicBoolean(false);
```

Comment:
What is the use of AtomicBoolean here? I didn't see it being used for any atomic operations in this class.